wandb/internal/internal_api.py

```diff
@@ -1477,6 +1477,8 @@ def create_artifact(self, artifact_type_name, artifact_collection_name, digest,
         if not is_user_created:
             run_name = run_name or self.current_run_id
 
+        if aliases is None:
+            aliases = []
```

Comment:
@annirudh I saw some weird stuff when testing.  Do we support the case of creating an artifact without any aliases?  For the new `Api().run.log_artifact()` method I didn't want to add the "latest" tag...

Reply:
We should, but it looks like it was bugged before your fix. Nice catch!